taxi/models.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ["name"]
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(to=Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(
+        to=settings.AUTH_USER_MODEL,
+        related_name="cars"
+    )

```

Comment:
Why does only one of these fields have a `related_name` redefined? It seems that this can make your code less intuitive to work with.